hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -356,9 +345,14 @@ void ISSWrapper::start() {
   insn_cnt_ = 0;
 }
 
-void ISSWrapper::edn_rnd_data(uint32_t edn_rnd_data[8]) {
+void ISSWrapper::edn_rnd_cdc_done() {
+  run_command("edn_rnd_cdc_done\n", nullptr);
+}
+
+void ISSWrapper::edn_step(uint32_t edn_rnd_data) {
   std::ostringstream oss;
-  oss << "edn_rnd_data " << wlen_val_to_hex_str(edn_rnd_data) << "\n";
+  oss << "edn_step " << std::hex << "0x" << std::setfill('0') << std::setw(1)
```

Comment:
Nit: I'd probably get rid of the `setfill` and `setw`: they won't do anything here, I don't think (because every value takes at least one character) and they aren't needed.